app/models/organization.rb

```diff
@@ -1,10 +1,12 @@
 class Organization < ApplicationRecord
+  update_index('locations#location') { locations }
+
   default_scope { order('id DESC') }
 
   has_many :locations, dependent: :destroy
   has_many :programs, dependent: :destroy
   has_many :contacts, dependent: :destroy
-  has_many :flags, dependent: :destroy
+  has_many :flags, dependent: :destroy, as: :resource
```

Comment:
@justuseapen This is needed as our `Flag` model is dependent on `resource_id` and `recource_type` columns which act like polymorphic.